database/schema.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE "judges"(
+    "judge_id" SMALLINT NOT NULL,
+    "name" TEXT NOT NULL,
+    "appointed" DATE NOT NULL,
+    "circuit_id" SMALLINT NULL,
+    "judge_type_id" BIGINT NOT NULL,
+    "gender" VARCHAR(1) NOT NULL
```

Comment:
i think gender and circuit_id at leat should allow NULL, maybe appointed too?

Reply:
Yes agreed, I'll change it now.